services/motion/client_test.go

```diff
@@ -29,6 +27,11 @@ import (
 	"go.viam.com/rdk/testutils/inject"
 )
 
+var (
+	testMotionServiceName  = motion.Named("motion1")
+	testMotionServiceName2 = motion.Named("motion2")
```

Comment:
[nit] `testMotionServiceName2` doesn't seem to be used, was this intended to be added to the `resource` map for a different injection?

Reply:
Never mind, found the pull from `motion_test.go` and the use in `server_test.go`

Reply:
Yeah I didn't add functionality to tests like this. Just moved stuff